.github/workflows/codeql-analysis.yml

```diff
@@ -1,10 +1,8 @@
 name: "CodeQL"
 
 on:
-  push:
-    branches:
-      - 'main'
   pull_request:
+  push:
     branches:
       - 'main'
```

Comment:
Most of these workflows were using:

```yaml
pull_request:
  branches:
    - main
```

Which means "run on PRs targeting `main`", preventing them from running on stacked PRs. That's not ideal, so this PR drops the `branches` filter from the `pull_request` trigger.